<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>480</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>640</width>
         <height>480</height>
        </size>
       </property>
       <property name="text">
        <string>视频播放框</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,4,4,4,4,1">
     <item>
      <widget class="QPushButton" name="bind">
       <property name="text">
        <string>绑定端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="unbind">
       <property name="text">
        <string>解除绑定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="act">
       <property name="text">
        <string>开始采集</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>本地播放</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>开启广播</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>您好</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="send">
       <property name="text">
        <string>发送消息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="broadcast">
       <property name="text">
        <string>广播消息</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
